File: fizzbuzz/fizzbuzz.py
from functools import reduce

def fizzbuzziness(num):
    """
    Return fizzbuzziness of a number

    Arguments:
    num -- an integer

    If num is a multiple of 3, 'Fizz'
    If num is a multiple of 5, 'Buzz'
    If num is a multiple of 3 and 5, 'FizzBuzz'
    If num is not a multiple of 3 or 5, outputs an empty string
    """

    tbl = { 'Fizz': num % 3 == 0, 'Buzz': num % 5 == 0 }
    trues = { k: v for k, v in tbl.items() if v }
    return reduce((lambda x, y: x + y), list(trues.keys()), '')

File: fizzbuzz/fizzbuzz.js
(() => {
  /**
   * @name fizzbuzziness
   * @description
   *   If num is a multiple of 3, 'Fizz'
   *   If num is a multiple of 5, 'Buzz'
   *   If num is a multiple of 3 and 5, 'FizzBuzz'
   *   If num is not a multiple of 3 or 5, outputs an empty string
   *
   * @param {Number} num: An integer number
   * @return {String} The 'FizzBuzziness' of num
  */
  'use strict'

  const fizzbuzziness = (num) => {
    return Object
      .keys(Object({ 'Fizz': num % 3 === 0, 'Buzz': num % 5 === 0 }))
      .filter(key => tbl[key])
      .reduce((acc, curr) => { return `${acc}${curr}` }, '')
  }
})()
